test/external-assertions/test.js

```diff
@@ -0,0 +1,13 @@
+import test from '@ava/test';
+
+import {fixture} from '../helpers/exec.js';
+
+test('node assertion ', async t => {
+	const result = await t.throwsAsync(fixture(['assert-failure.js']));
+	t.snapshot(result.stdout);
```

Comment:
Looks like the tests are failing because the error message contains carriage returns. IIRC AVA only emits line feeds which would explain why there's no automatic handling of the discrepancy.

```suggestion
	t.snapshot(result.stdout.replace(/\r/g, ''));
```

(I'd suggest `replaceAll('\r', '')` but as of right now AVA still targets Node.js 14 which doesn't support that.)